packages/svelte/src/compiler/migrate/index.js

```diff
@@ -729,9 +744,53 @@ const template = {
 		}
 		next();
 	},
+	SvelteSelf(node, { state, next }) {
+		const source = state.str.original.substring(node.start, node.end);
+		if (!state.filename) {
+			const indent = guess_indent(source);
+			state.str.prependRight(
+				node.start,
+				`<!-- @migration-task: migrate this by hand or call the migrate function with the filename of this file -->\n${indent}`
```

Comment:
Better to give clear instructions, not sure if people know what we mean by "migrate with the filename"
```suggestion
				`<!-- @migration-task: svelte:self is deprecated, import this Svelte file into itself instead -->\n${indent}`
```